### ISSUE TITLE
[fr] AMOLED theme

### ISSUE THREAD

**Issue by throwaway242685** [jjewuz/JustWeather#4]
hi, I was wondering if there were any plans to implement an AMOLED theme?

**Comment by jjewuz** [comment]
Yes, of course I can do it, there was just no demand. If this is what you need, I will try to do it in the near future
